tools/debian/convert_debian.py

```diff
@@ -34,97 +36,179 @@
 # e.g. [buster] - xz-utils 5.2.4-1+deb10u1
 VERSION_PATTERN = re.compile(r'\[(.*?)\]\s*-\s*([^\s]+)\s*([^\s]+)')
 
+# e.g. <define-tag moreinfo>\n Some html here \n</define-tag>
+WML_DESCRIPTION_PATTERN = re.compile(r'<define-tag moreinfo>((?:.|\n)*)</define-tag>', re.MULTILINE)
```

Comment:
Can you make all lines <= 80 columns? 

You can use `yapf` and `pylint` to fix/find these issues.